### ISSUE TITLE
Prereleases 4.0.8-preview801 and upwards do not properly contain Publish-PerTenantExtensionApps function.

### ISSUE THREAD

**Issue by Jupentti1** [microsoft/navcontainerhelper#2824]
PLEASE DO NOT INCLUDE ANY PASSWORDS OR TOKENS IN YOUR ISSUE!!!

**Describe the issue**
Prereleases 4.0.8-preview801 and upwards do not properly contain Publish-PerTenantExtensionApps function.

**Scripts used to create container and cause the issue**
```
Write-Host "Installing BcContainerHelper"
Install-Module BcContainerHelper -Force -AllowPrerelease
Write-Host "Publishing to tenant"
Publish-PerTenantExtensionApps -useNewLine `
 -ClientID "$(PublisherAppClientID)" `
 -ClientSecret "$(PublisherAppClientSecret)" `
 -tenantId "$(TenantId)" `
 -environment "$(Environment)" `
 -companyName "$(CompanyName)" `
 -appFiles @(Get-Item "_Develop/output/Apps/*.app" | % { $_.FullName })
```

**Full output of scripts**
```
2022-12-14T14:35:59.7628876Z Installing BcContainerHelper
2022-12-14T14:37:46.0153432Z Publishing to tenant
2022-12-14T14:37:46.1341745Z Publish-PerTenantExtensionApps : The term 'Publish-PerTenantExtensionApps' is not recognized as the name of a cmdlet, 
2022-12-14T14:37:46.1343870Z function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the 
2022-12-14T14:37:46.1344979Z path is correct and try again.
2022-12-14T14:37:46.1346268Z At D:\a\_temp\2ef56dd1-4109-4b15-94bf-bccc32e965f5.ps1:7 char:1
2022-12-14T14:37:46.1347167Z + Publish-PerTenantExtensionApps -useNewLine `
2022-12-14T14:37:46.1347845Z + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2022-12-14T14:37:46.1350715Z     + CategoryInfo          : ObjectNotFound: (Publish-PerTenantExtensionApps:String) [], ParentContainsErrorRecordExc 
2022-12-14T14:37:46.1351761Z    eption
2022-12-14T14:37:46.1352360Z     + FullyQualifiedErrorId : CommandNotFoundException
2022-12-14T14:37:46.1352977Z  
2022-12-14T14:37:46.2523344Z ##[error]PowerShell exited with code '1'.
...
```

**Screenshots**

**Additional context**
- does it happen all the time? Yes
- did it use to work? Yes and still does without the -AllowPrerelease option

**Comment by freddydk** [comment]
If you add Import-Module after Install-Module - does it work then?

**Comment by Jupentti1** [comment]
It is part of the CD pipeline so would this have any effect?

**Comment by freddydk** [comment]
Just modify the code:
```
Write-Host "Installing BcContainerHelper"
Install-Module BcContainerHelper -Force -AllowPrerelease
Import-Module BcContainerHelper -DisableNameChecking
Write-Host "Publishing to tenant"
Publish-PerTenantExtensionApps -useNewLine `
```
If that works - then I can figure out how to remove the need for that.

**Comment by Jupentti1** [comment]
It worked with modified code, thanks.

**Comment by freddydk** [comment]
Will fix this before shipping 4.0.8 - but this workaround can be used and will also work after the fix.

**Comment by freddydk** [comment]
Shipped in 4.0.9
